### PR TITLE
Create 2022-03-30.cross-compilation.md

### PR DESCRIPTION
**Pull request by christinerose**
Scheduled to be published on 30 March 2022 according to the [MirageOS Release Comms Plan](https://hackmd.io/3M7iWVs3Q4-T0bBb0b0maQ).

If there are any edits, please let me look at it again for grammar/formatting consistency. Thanks!

### CODE REVIEW

**Comment by samoht** [comment]
@TheLortex are the pins still necessary?

**Comment by dinosaure** [comment]
> The big issue I have encountered is a compilation error when using Dune 3.

Which error?

File: data/blog/2022-03-30.cross-compilation.md [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+
+(alias
+  (name default)
+  (enabled_if (= %{context_name} "solo5"))
+  (deps (alias_rec all)))
+```
+
+
+
+
```
Comment (samoht) [inline data/blog/2022-03-30.cross-compilation.md:193]:
```suggestion
```

**Comment by TheLortex** [comment]
I have created an issue @dinosaure : https://github.com/mirage/mirage-crypto/issues/153

**Pull request merged**